### ISSUE TITLE
Modified to 3D network

### ISSUE THREAD

**Issue by XIANYUNYEHE-DEL** [keyu-tian/SparK#46]
Hi, I am very interested in your research and want to use it for 3D dataset. I want to know where this code and file needs to be modified to adapt to the 3D network. I would thanks for your reply.

**Comment by keyu-tian** [comment]
Thanks for your interest! Since 3d training is quite different from 2d image training, I would suggest to start form a 3d pretraining codebase, then modify it by adding the loss function and the UNet-style decoder of SparK into it.
